dotnet/src/Connectors/Connectors.AI.OpenAI/TextCompletion/AzureTextCompletion.cs

```diff
@@ -68,5 +68,21 @@ public sealed class AzureTextCompletion : AzureOpenAIClientBase, ITextCompletion
     {
         return this.InternalCompletionStreamAsync(text, requestSettings, cancellationToken);
     }
+
+    public IAsyncEnumerable<ITextCompletionStreamingResult> CompleteMultiStreamAsync(
+        string text,
+        CompleteRequestSettings requestSettings,
+        CancellationToken cancellationToken = default)
+    {
+        return this.InternalMultiCompletionStreamAsync(text, requestSettings, cancellationToken);
+    }
+
+    public Task<IReadOnlyList<ITextCompletionResult>> CompleteMultiAsync(
+        string text,
+        CompleteRequestSettings requestSettings,
+        CancellationToken cancellationToken = default)
+    {
+        return this.InternalCompleteTextMultiAsync(text, requestSettings, cancellationToken);
+    }
```

Comment:
One thought: consider using overloads for CompleteAsync:
- one with a resultsPerPrompt arg that returns a collection, and 
- one without that returns a single instance. Same for CompleteStreamAsync().

Then we don't need the "Multi" methods, and `ResultsPerPrompt` need not be in the CompleteRequestSettings, but rather can be specified on a per-usage basis (i.e. not hard-coded in a prompt's config.json)

Reply:
Yes that's the final intention here, I used other methods just to validate the approach. Will move those internals to be all multi capable with no dupes.

Reply:
I moved forward with the implementation and added the multiple results as the standard interfaces behavior. 

The completionsettings has the property but this is not currently exposed to the json file, reading from a json file currently will default to 1 ResultsPerPrompt.